### ISSUE TITLE
The results of VOC2012 val dataset

### ISSUE THREAD

**Issue by zwy1996** [kazuto1011/deeplab-pytorch#101]
Hi, Thanks  for your excellent job.

I am very sorry to bother you about the questions. When I ran the instructions as you wrote on the readme and I had no change, but I  got miou=85%, Was the results normal?
{
    "Class IoU": {
        "0": 0.9626663940414879,
        "1": 0.9117191232908595,
        "2": 0.4307306602142222,
        "3": 0.9116816722148997,
        "4": 0.8235518201472711,
        "5": 0.8892424282764084,
        "6": 0.9625425319789785,
        "7": 0.9044427683338813,
        "8": 0.9485119216054932,
        "9": 0.6063125420512411,
        "10": 0.9372268530649261,
        "11": 0.7883814139533765,
        "12": 0.9294599576731154,
        "13": 0.915489240578804,
        "14": 0.8856504895476909,
        "15": 0.8937371321051956,
        "16": 0.7560997924772654,
        "17": 0.9276748086051673,
        "18": 0.7849737860274428,
        "19": 0.938800648058414,
        "20": 0.8337808227300536
    },
    "Frequency Weighted IoU": 0.941278432096473,
    "Mean Accuracy": 0.9214160613860314,
    "Mean IoU": 0.8544131812845805,
    "Pixel Accuracy": 0.9683390562687486
}
Thank you!

**Comment by kazuto1011** [comment]
Did you train by yourself or just evaluate with the released weights?
I think issue #63 may be relevant to yours.

**Comment by zwy1996** [comment]
Thank you very much!
I change the pre-trained model, and I can get the results as you now.
